src/cpp/core/json/Json.cpp

```diff
@@ -155,6 +156,56 @@ std::string writeFormatted(const Value& value)
    return json_spirit::write_formatted(value);
 }
 
+Error writeCsv(const Array& array, std::ostream& os)
+{
+   std::size_t nrow = array.size();
+   if (nrow == 0)
+      return Success();
+   
+   Value value;
+   Object object;
+   
+   for (std::size_t i = 0; i < nrow; ++i)
+   {
+      // get array element as object
+      value = array[i];
+      if (!isType<Object>(value))
+         return Error(errc::ParamInvalid, ERROR_LOCATION);
+      object = value.get_obj();
+      
+      if (object.size() == 0)
+         continue;
+      
+      Object::const_iterator it;
+
+      // first object: write names
+      if (i == 0)
+      {
+         it = object.begin();
+         json::write(it->first, os);
+         for (++it; it != object.end(); ++it)
+         {
+            os << ",";
+            json::write(it->first, os);
+         }
+         os << "\n";
```

Comment:
We can indeed remove it (it was an artefact of code assuming we might need to parse the `chunks.json` in the `rmarkdown` package)